typed-racket-lib/typed-racket/typecheck/check-below.rkt

```diff
@@ -1,19 +1,20 @@
 #lang racket/base
 
-(require "../utils/utils.rkt"
-         racket/match (prefix-in - (contract-req))
-         racket/format
+(require racket/format
+         racket/match
+         (prefix-in - (contract-req))
```

Comment:
Seems this line is busted now because `contract-req` is a require transformer that came from `"../utils/utils.rkt"`.

Reply:
The way this is handled in most formatters / linters is via annotation. E.g., users could add `#;#[[resyntax ignore-next]]` right before the `(require ...)`, and the whole `(require ...)` should not be refactored.

Reply:
In theory Resyntax has the whole binding structure available, so it could see that `contract-req` is imported from one of the modules in this require form. That could be a cue to avoid tidying this one.

Reply:
I see that check syntax doesn't add this arrow so maybe there is an improvement there that resyntax could then use?

Reply:
There were some improvements in march but I think there's more that can be done.

Reply:
Also one way to fix this specific code is to move the `"../utils/utils.rkt"` import into a separate `require` form, which I generally think is a good idea when requiring modules that provide require or provide transformers.